jest/setupTests.ts

```diff
@@ -14,3 +16,8 @@ configure({adapter: new Adapter()});
 // add convenience variables to the global context
 (global as any).verifyComponent = verifyComponent;
 (global as any).jest = jest;
+
+beforeEach(() => {
+  setUniqueSeed('a');
+  resetUniqueIdCount();
+});
```

Comment:
This isn't actually needed for our tests, but it is nice to have something to point to as an example for teams that need this setup.

Reply:
I'm sure teams are gonna wanna use this, nice addition

Reply:
Should something like this go into the testing docs for reference?

Reply:
nvm you documented this pretty well in the readme